pages/admin/request/create-new.tsx

```diff
@@ -0,0 +1,845 @@
+import { formatDateYYYYMMDD } from '@lib/utils/format';
+
+/** Create New APP page */
+export default function CreateNew() {
+  const [step, setStep] = useState<RequestFlowPageState>(
+    RequestFlowPageState.SelectingPermitHolderPage
+  );
+  const [permitHolderExists, setPermitHolderExists] = useState(true);
+
+  const [applicantId, setApplicantId] = useState<number | null>(null); // RCD user id
+
+  // General information
+  const [permitHolderInformation, setPermitHolderInformation] =
+    useState<NewApplicationPermitHolderInformation>({
+      firstName: '',
+      middleName: '',
+      lastName: '',
+      dateOfBirth: '',
+      gender: 'MALE',
+      otherGender: '',
+      email: '',
+      phone: '',
+      receiveEmailUpdates: false,
+      addressLine1: '',
+      addressLine2: '',
+      city: '',
+      postalCode: '',
+    });
+  // Physician assessment
+  const [physicianAssessment, setPhysicianAssessment] = useState<PhysicianAssessment>({
+    disability: '',
+    patientCondition: null,
+    permitType: null,
+    physicianCertificationDate: new Date().toLocaleDateString('en-CA'),
+  });
+  // Doctor information
+  const [doctorInformation, setDoctorInformation] = useState<DoctorFormData>({
+    firstName: '',
+    lastName: '',
+    mspNumber: null,
+    phone: '',
+    addressLine1: '',
+    addressLine2: '',
+    city: '',
+    postalCode: '',
+  });
+  // Guardian/POA information
+  const [guardianInformation, setGuardianInformation] = useState<GuardianInformation>({
+    omitGuardianPoa: false,
+    firstName: '',
+    middleName: '',
+    lastName: '',
+    relationship: '',
+    phone: '',
+    addressLine1: '',
+    addressLine2: '',
+    city: '',
+    postalCode: '',
+    // TODO: poaFormUrl work with file upload
+    poaFormUrl: '',
+  });
+  // Additional questions
+  const [additionalQuestions, setAdditionalQuestions] = useState<AdditionalInformationFormData>({
+    usesAccessibleConvertedVan: null,
+    accessibleConvertedVanLoadingMethod: null,
+    requiresWiderParkingSpace: null,
+    requiresWiderParkingSpaceReason: null,
+    otherRequiresWiderParkingSpaceReason: null,
+  });
+  // Payment information
+  const [paymentDetails, setPaymentDetails] = useState<PaymentInformationFormData>({
+    paymentMethod: null,
+    donationAmount: '',
+    shippingAddressSameAsHomeAddress: false,
+    shippingFullName: '',
+    shippingAddressLine1: '',
+    shippingAddressLine2: '',
+    shippingCity: '',
+    shippingProvince: 'BC',
+    shippingCountry: '',
+    shippingPostalCode: '',
+    billingAddressSameAsHomeAddress: false,
+    billingFullName: '',
+    billingAddressLine1: '',
+    billingAddressLine2: '',
+    billingCity: '',
+    billingProvince: 'BC',
+    billingCountry: '',
+    billingPostalCode: '',
+  });
+
+  // Toast message
+  const toast = useToast();
+
+  // Router
+  const router = useRouter();
+
+  // Reset all fields when application is discarded
+  const resetAllFields = () => {
```

Comment:
Good catch to add this function. Perhaps we can add the initial values (eg. `INITIAL_PERMIT_HOLDER_INFORMATION`, `INITIAL_PHYSICIAN_ASSESSMENT`, etc) to the tools file to reduce the amount of code/redundancy here